airflow/jobs/backfill_job.py

```diff
@@ -782,6 +784,28 @@ def _execute(self, session=None):
                 return
             dagrun_infos = [DagRunInfo.interval(dagrun_start_date, dagrun_end_date)]
 
+        dag_with_subdags_ids = [d.dag_id for d in self._get_dag_with_subdags()]
+        running_dagruns = DagRun.find(
+            dag_id=dag_with_subdags_ids,
+            execution_start_date=self.bf_start_date,
+            execution_end_date=self.bf_end_date,
+            no_backfills=True,
+            state=State.RUNNING,
```

Comment:
Please prefer `DagRunState`. Same applies for other changes. (For `State.NONE`, simply use `None` instead.)

Reply:
updated